src/core/matrix.c

```diff
@@ -95,6 +95,20 @@ igraph_error_t igraph_matrix_complex_fprint(const igraph_matrix_complex_t *m,
     return IGRAPH_SUCCESS;
 }
 
+/**
+ * \ingroup matrix
+ * \function igraph_matrix_complex_real
+ * \brief Gives the real part of a complex matrix.
+ *
+ * \param v Pointer to a complex matrix.
+ * \param real Pointer to an initialized matrix. The result will be stored here.
+ * \return Error code.
+ *
+ * Time complexity: O(n),
+ * n is the
+ * number of elements in the matrix.
+ */
+
 igraph_error_t igraph_matrix_complex_real(const igraph_matrix_complex_t *v,
```

Comment:
Let's call the matrix `m` instead of `v`, both in the argument list and in the docs. Is it easy for you to do this with your IDE? If not, I can help. Most IDEs have a "rename" feature.

Reply:
vim handled it quite well once I understood how to do it :)